modules/standards/src/standard-entity-definition.ts

```diff
@@ -6,24 +6,22 @@ import {
 
 type AuthorizationSetting = {};
 
-// @ts-ignore @TODO: implement this class
 export class StandardEntityDefinition implements EntityDefinition {
   authorizationSetting: AuthorizationSetting;
 
   constructor(authorizationSetting: AuthorizationSetting) {
     this.authorizationSetting = authorizationSetting;
   }
 
-  async authorization(
+  async authorize(
     reqData: GeneralRequestData,
     session: Session | null | undefined
   ): Promise<boolean> {
-    // eslint-disable-line no-unused-vars
     // TODO
     return false;
   }
 
-  async validation(
+  async validate(
     reqData: GeneralRequestData,
```

Comment:
`GeneralRequestData `  --> `UserEntityRequestData `  ?

Reply:
I think `GeneralRequestData` is correct because the arg of `reqData` is GeneralRequestData in EntityDefinition.
I guess `StandardUserDefinition` is limited to users, so the `reqData` changed to `UserEntityRequestData`.
https://github.com/phenyl-js/phenyl/blob/master/modules/interfaces/src/entity-definition.ts#L11